docs/un-supported-patch-properties.md

```diff
@@ -30,4 +30,4 @@ July 07, 2022
 
 ## How to fix the violation
 
-Considering removing the name, location, or type in the patch request body.
+Consider either removing the top level properties - id, name and type, from the patch request body or mark them as readOnly. For the top level location property for tracked resources, consider either removing it from the request body of the Patch operation or mark it as immutable using the x-ms-mutability property and values as "create" and "read".
```

Comment:
What does `for tracked resources` mean here? Can there be a top-level "location" property for untracked resources?

Reply:
"location" property can only exist for tracked resources. Tracked resources are resources that are stored by ARM in its registry. A resource is a representation of an object that follows the REST semantics

Reply:
I edited the description to make it convey the meaning better